File: 10_week_2/nth_to_last.js
// return the element that is N-from-array's-end.

function nth_to_last(arr, n) {
	if ((arr.length - n < 0) || (n <= 0)) {
		return null;
	}
	return arr[arr.length - n];
}


var arr = [1, 2, 3, 4, 5, 6];
var result = nth_to_last(arr, 2);
console.log(result);

File: 10_week_2/nth_to_last.py
# return the element that is N-from-array's-end.

def nth_to_last(arr, n):
	if len(arr) - n < 0 or n <= 0:
		return None;
	return arr[len(arr) - n]

arr = [1, 2, 3, 4, 5, 6]
result = nth_to_last(arr, 2)
print(result)
